fdbserver/KeyValueStoreRocksDB.actor.cpp

```diff
@@ -248,8 +288,8 @@ struct RocksDBKeyValueStore : IKeyValueStore {
 				}
 				s = cursor->status();
 			}
-
-			if (!s.ok()) {
+			auto s = cursor->status();
```

Comment:
This might be a rebase error.  The then and else branch above call `s = cursor->status()` as their last line.  I didn't intentionally invoke it twice.  I  probably tried to  delete the two calls above and replace them with this call.

[ edit: also, this won't compile, since `auto s` would mask `rocksdb::Status s` above.